spec/factories.rb

```diff
@@ -225,8 +225,8 @@
       description { Forgery(:basic).text }
     end
 
-    trait :with_children do
-      children_geographical_areas { [attributes_for(:geograpical_area).stringify_keys] }
+    initialize_with do
```

Comment:
This enables us to be able to call `build(:geographical_area)` rather than always passing the attributes directly to the constructor of the `GeographicalArea` class with `GeographicalArea.new(attributes_for(:geographical_area))`